crates/wasi-http/src/struct.rs

```diff
@@ -8,22 +8,68 @@ pub struct Stream {
     pub data: BytesMut,
 }
 
+impl crate::types::Method {
+    pub fn new(m: &hyper::Method) -> Self {
```

Comment:
Idiom: this constructor is better phrased as an `impl From<hyper::Method for crate::types::Method`